addOns/tokengen/src/main/java/org/zaproxy/zap/extension/tokengen/TokenPanel.java

```diff
@@ -263,6 +263,7 @@ private JToggleButton getPauseScanButton() {
                     new ImageIcon(getClass().getResource("/resource/icon/16/131.png")));
             pauseScanButton.setEnabled(false);
             pauseScanButton.addActionListener(e -> pauseScan());
+            // something is amiss
```

Comment:
Why are you futzing with token gen?

Reply:
We're trying to figure out other places that pause the Scan to try and see what can imitated with the resumescan feature.  So we have a couple comments to mark things of interest